rke/k8s_rke_system_images.go

```diff
@@ -8374,6 +8374,49 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		"v1.25.2-rancher1-1": {
```

Comment:
need comment on top of this, `Enabled in Rancher vx.x.x`

Reply:
comment is added